package.json

```diff
@@ -20,9 +21,9 @@
     "react-i18next": "7.3.1",
     "react-redux": "5.0.6",
     "react-router-dom": "4.2.2",
-    "react-scripts": "1.0.17",
+    "react-scripts": "2.1.1",
     "react-select": "1.0.0-rc.10",
-    "react-svg": "2.1.22",
+    "react-svg": "^2.2.18",
```

Comment:
Why are these being setup to always use the latest version? In the past that has proven to be a bit unsafe.

Reply:
Let me check... I thought I fixed those in a second commit.